common/app/Panes/Panes.jsx

```diff
@@ -44,38 +45,75 @@ const propTypes = {
   render: PropTypes.func.isRequired
 };
 
+let quoteObj = fetchRandomQuote();
+
 export class Panes extends PureComponent {
   componentDidMount() {
     this.props.panesMounted();
   }
+  componentDidUpdate(prevProps) {
+    if (prevProps.panes.length === 0 && this.props.panes.length === 1) {
+      const currentQuote = quoteObj.quote;
+      let newQuoteObj = fetchRandomQuote();
+      while (currentQuote === newQuoteObj.quote) {
+        newQuoteObj = fetchRandomQuote();
+      }
+      quoteObj = newQuoteObj;
+    }
+  }
+
   renderPanes() {
     const {
       render,
       panes
     } = this.props;
-    return panes.map(({ name, left, right, dividerLeft }) => {
-      const divider = dividerLeft ?
-        (
-          <Divider
-            key={ name + 'divider' }
-            left={ dividerLeft }
-            name={ name }
-          />
-        ) :
-        null;
+    if (panes.length > 0) {
+      return panes.map(({ name, left, right, dividerLeft }) => {
+        const divider = dividerLeft ?
+          (
+            <Divider
+              key={ name + 'divider' }
+              left={ dividerLeft }
+              name={ name }
+            />
+          ) :
+          null;
+
+        return [
+          <Pane
+            key={ name }
+            left={ left }
+            right={ right }
+            >
+            { render(name) }
+          </Pane>,
+          divider
+        ];
+      }).reduce((panes, pane) => panes.concat(pane), [])
+        .filter(Boolean);
+    } else {
+      return this.renderQuote();
+    }
+  }
 
-      return [
-        <Pane
-          key={ name }
-          left={ left }
-          right={ right }
-          >
-          { render(name) }
-        </Pane>,
-        divider
-      ];
-    }).reduce((panes, pane) => panes.concat(pane), [])
-      .filter(Boolean);
+  renderQuote() {
+    return (
+      <div className='outer-flex-container'>
+        <div className='flex-length' />
+        <div className='quote-block'>
+          <div>
+            <h2 className='quote-style'>&ldquo;{quoteObj.quote}&rdquo;</h2>
+            <h2 className='author-style'><i>&mdash;{quoteObj.author}</i></h2>
+          </div>
+        </div>
+        <div className='logo-block'>
```

Comment:
![image](https://user-images.githubusercontent.com/18572518/38249618-2e937130-3744-11e8-9660-a49d08ec2978.png)

Reply:
That's right, I hadn't considered night mode at all! I can remove the logo -- seems redundant with in the nav, anyway. It'll also give me a chance to simplify the Flexbox layout a bit. I'll submit an update soon.